### ISSUE TITLE
Missing header Invoice PDF reports on Odoo v13.0 

### ISSUE THREAD

**Issue by bakatasolutionssl** [odoo/odoo#72534]
Impacted versions: 13.0 Community (13.0.20210622)

Steps to reproduce: 

Starting from invoicing, click on an invoice, then click on "preview", then click on Print, and in the PDF report, the invoice header will be missing

Current behavior: Invoice's header does not appear in PDF reports, but it appears in the invoice viewer

Expected behavior: Invoice's header appear in PDF reports just like it appears in the invoice viewer

Video/Screenshot link (optional):

**Invoice viewer:**

![image](https://user-images.githubusercontent.com/66942845/122925626-ec503c80-d35e-11eb-9014-842cbbe7a04b.png)

**Invoice PDF report:**

![image](https://user-images.githubusercontent.com/66942845/122925778-130e7300-d35f-11eb-98ee-7d591ed4d7e9.png)

Support ticket number submitted via odoo.com/help (optional):

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by bakatasolutionssl** [comment]
Hi @pedrobaeza ,

Thank you for response.

We have a virtual machine with Odoo 13 Community (13.0.20210622), without third party modules, and with the demo data to test and we experimented this issue.

It does not happen on runbot but the first difference betweet runbot and our virtualbox machine is the domain. 
We enter to Odoo through IP:8069 (for example 192.168.1.54:8069). 

Could it be the cause? Does Odoo need to be accessed through a domain and a reverse proxy  to be able to print PDF reports correctly? Or it needs a special configuration when it is not being accessed through a domain?



All the modules we have installed:

![image](https://user-images.githubusercontent.com/66942845/122933270-633d0380-d366-11eb-80cc-f45812f9ec62.png)

System Parameters:

![image](https://user-images.githubusercontent.com/66942845/122933364-7ea80e80-d366-11eb-94fd-edcf783d3be4.png)

Our /etc/odoo/odoo.conf

![image](https://user-images.githubusercontent.com/66942845/122933533-a303eb00-d366-11eb-8fba-bd24d21c837a.png)

We don't have domain and reverse proxy.

Thank you.
